java-sdk/connectors/security-filtering-connector/src/main/java/com/lucidworks/fusion/connector/plugin/config/SecurityFilteringConfig.java

```diff
@@ -21,7 +21,7 @@
   )
   Properties properties();
   
-  interface Properties extends RandomContentConfig.Properties, SecurityTrimmingProperties {
+  interface Properties extends RandomContentConfig.Properties, SecurityTrimmingConfig {
```

Comment:
It should be validated that this change is not causing a regression problem, this is related to APOLLO-17245

Reply:
@wsolaligue Running pytest against this branch passes without issues.  Also verified in running that the access control hierarchy collection is created.